docker/cmsmon-rucio-ds/Dockerfile

```diff
@@ -33,12 +33,13 @@ RUN yum remove -y hbase-bin-2.3 && \
     rm -rf stomp-v700 && \
     svn export https://github.com/dmwm/CMSMonitoring.git/branches/master/src/python/CMSMonitoring && \
     zip -r CMSMonitoring.zip CMSMonitoring/* && \
-    git clone https://github.com/mrceyhun/CMSSpark.git && \
-    cd CMSSpark && git checkout f-rucio_datasets_daily_stats && \
+    git clone https://github.com/dmwm/CMSSpark.git && \
     yum clean all &&  rm -rf /var/cache/yum && \
     rm -f /usr/bin/python && ln -s /usr/bin/python3.6 /usr/bin/python && \
-    hadoop-set-default-conf.sh analytix && \
-    source hadoop-setconf.sh analytix 3.2 spark3
-
+    hadoop-set-default-conf.sh analytix && source hadoop-setconf.sh analytix 3.2 spark3
 
 WORKDIR $WDIR
+
+# Run crond
```

Comment:
I doubt it is correct approach here since you put hard-coded logic (of crontab.txt) into image. This implies that image will only work if all proper parameters are in place which are not know a-priory for someone who will take the image. I still prefer to create config map in k8s manifest file which will specify all parameters how to run the cron.